db/migrations/20230209072625_create_product_types_table.sql

```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+CREATE TABLE product_types (
+  id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  type VARCHAR(255) NOT NULL,
+  CONSTRAINT categories_name_ukey UNIQUE (type)
```

Comment:
해당 제약조건의 이름이 테이블명과 컬럼명을 충분히 담고 있지 않은 것 같습니다.
아래와 같이 수정해주세요.
```suggestion
CREATE TABLE product_types (
  id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
  type VARCHAR(255) NOT NULL,
  CONSTRAINT product_types_type_ukey UNIQUE (type)
```